pom.xml

```diff
@@ -411,53 +358,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-shade-plugin</artifactId>
-        <version>3.5.1</version>
-        <executions>
-          <execution>
-            <phase>package</phase>
-            <goals>
-              <goal>shade</goal>
-            </goals>
-            <configuration>
-              <transformers>
-                <transformer implementation="org.apache.maven.plugins.shade.resource.DontIncludeResourceTransformer">
-                  <resource>.proto</resource>
-                  <resource>checkstyle.xml</resource>
-                </transformer>
-                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
-                <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer">
-                  <addHeader>true</addHeader>
-                </transformer>
-              </transformers>
-              <relocations>
-                <relocation>
-                  <pattern>com.google</pattern>
-                  <shadedPattern>com.spotify.confidence.shaded.com.google</shadedPattern>
-                </relocation>
-                <relocation>
-                  <pattern>io.grpc</pattern>
-                  <shadedPattern>com.spotify.confidence.shaded.io.grpc</shadedPattern>
-                </relocation>
-              </relocations>
-              <artifactSet>
-                <excludes>
-                  <exclude>dev.openfeature:sdk</exclude>
-                  <exclude>org.slf4j:slf4j-api</exclude>
-                  <exclude>javax.annotation:javax.annotation-api</exclude>
-                  <exclude>org.checkerframework:checker-qual</exclude>
-                  <exclude>com.google.code.findbugs:jsr305</exclude>
-                  <exclude>com.google.android:annotations</exclude>
-                  <exclude>org.codehaus.mojo:animal-sniffer-annotations</exclude>
-                  <exclude>io.perfmark:perfmark-api</exclude>
-                </excludes>
-              </artifactSet>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
These needs to be moved to the confidence pom.

Reply:
Shading grpc classes from `Confidence` currently breaks the `OpenFeature` part, since the latter now doesn't use shaded grpc dependencies. Since the shading is confusing, I wonder if we could remove it until it's clear that we actually need it

Reply:
but why would openfeature need to know about grpc?